content/body/textbox.php

```diff
@@ -100,12 +100,12 @@
 <?php includeStats([
     "isForNewBuilds" => false,
     "comment" =>
-        "Recommended only if you needed to create a JavaScript WYSIWYG editor.",
+        "It is recommended only if you need to create a JavaScript WYSIWYG editor.",
 ]); ?>
 
 <p>
-  Keep in mind that if you use this in a form, none of the nice free form functionality (e.g.: HTML5 validation,
-  inclusion of data when submitting a form in an HTTP request, etc), won't work. These example do, however, show up in
+  Keep in mind that if you use this in a form, none of the nice free-form functionality (e.g., HTML5 validation,
```

Comment:
I don't think we usually included commas after "e.g.". Do we need it here?
```suggestion
  Keep in mind that if you use this in a form, none of the nice free-form functionality (e.g. HTML5 validation,
```